examples/viewer_fps_label.py

```diff
@@ -0,0 +1,17 @@
+"""
+Display a 3D volume and the fps label
+"""
+import numpy as np
+import napari
+
+
+def update_fps(fps):
+    """Update fps"""
+    viewer.label.text = "%1.1f FPS" % fps
```

Comment:
Maybe using an f-string would make this more up to date?

`viewer.label.text = f"%{fps:1.1f} FPS"`